packages/amplify/amplify_flutter/README.md

```diff
@@ -2,9 +2,13 @@
 
 The top level module for Amplify Flutter.
 
+⚠️ **Amplify Flutter v0 is now in Maintenance Mode until July 19th, 2024. This means that we will continue to include updates to ensure compatibility with backend services and security. No new features will be introduced in v0.**
+
+Please use the latest version (v1) of Amplify Flutter. If you are currently using v0, follow [these instructions](https://docs.amplify.aws/lib/project-setup/upgrade-guide/q/platform/flutter/) to upgrade to v1.
+
 ## Category / Platform Support
 
-### Stable Release
+### v0
```

Comment:
do we want to flip the versions here same as above for consistancy?

Reply:
done, ty